x-pack/plugins/enterprise_search/public/applications/esre/components/esre_guide/rank_aggregation_section.tsx

```diff
@@ -18,69 +18,80 @@ import { EsreGuideAccordion } from './esre_guide_accordion';
 import { LinearCombinationPanel } from './linear_combination_panel';
 import { RrfRankingPanel } from './rrf_ranking_panel';
 
-export const RankAggregationSection: React.FC = () => (
-  <EuiFlexGroup alignItems="center">
-    <EuiFlexItem grow={4}>
-      <EuiFlexGroup direction="column" gutterSize="s" justifyContent="flexStart">
-        <EuiFlexItem grow={false}>
-          <EuiTitle>
-            <h2>
-              <FormattedMessage
-                id="xpack.enterpriseSearch.esre.rankAggregationSection.title"
-                defaultMessage="Use a rank aggregation method"
-              />
-            </h2>
-          </EuiTitle>
-        </EuiFlexItem>
-        <EuiFlexItem grow={false}>
-          <EuiText>
-            <p>
-              <FormattedMessage
-                id="xpack.enterpriseSearch.esre.rankAggregationSection.description"
-                defaultMessage="Optional methods for fusing or combining different rankings to achieve better overall ranking performance."
-              />
-            </p>
-          </EuiText>
-        </EuiFlexItem>
-      </EuiFlexGroup>
-    </EuiFlexItem>
-    <EuiFlexItem grow={6}>
-      <EuiFlexGroup direction="column">
-        <EuiFlexItem grow={false}>
-          <EsreGuideAccordion
-            id="rrfRankingAccordion"
-            icon={rrfRankingIllustration}
-            title={i18n.translate('xpack.enterpriseSearch.esre.rrfRankingAccordion.title', {
-              defaultMessage: 'RRF hybrid ranking',
-            })}
-            description={i18n.translate(
-              'xpack.enterpriseSearch.esre.rrfRankingAccordion.description',
-              {
-                defaultMessage: 'Intelligently combines rankings without configuration',
-              }
-            )}
-          >
-            <RrfRankingPanel />
-          </EsreGuideAccordion>
-        </EuiFlexItem>
-        <EuiFlexItem grow={false}>
-          <EsreGuideAccordion
-            id="linearCombinationAccordion"
-            icon={linearCombinationIllustration}
-            title={i18n.translate('xpack.enterpriseSearch.esre.linearCombinationAccordion.title', {
-              defaultMessage: 'Linear combination',
-            })}
-            description={i18n.translate(
-              'xpack.enterpriseSearch.esre.linearCombinationAccordion.description',
-              {
-                defaultMessage: 'Weighted results from multiple rankings',
-              }
-            )}
-          >
-            <LinearCombinationPanel />
-          </EsreGuideAccordion>
-        </EuiFlexItem>
-      </EuiFlexGroup>
-    </EuiFlexItem>
-  </EuiFlexGroup>
-);
+export const RankAggregationSection: React.FC = () => {
```

Comment:
Should we have a unit test for this? It's only EUI and React, but I feel it would be good to test `RankAggregationSection`'s accordion behavior (simulate click on the toggle button, verify the other one closes).

If agreed, please suggest the enzyme functions / component structure to use. I guess `shallow()` won't work because the children are not rendered, and I haven't worked with `dive()` or `mount()` yet.

Reply:
You can unit test, it's not necessary. You'd need to use `mount()` to mount the full rendering tree, `dive` helps to locate the right element.

Reply:
I'm struggling with `mount()`, it's giving me this error:
```
  ● SemanticSearchSection › renders

    [KEA] Can not access "values" on logic "enterprise_search.kibana_logic" because it is not mounted!
```
Not sure what to mock, I can't find references in the code. I'll skip adding unit tests.

Reply:
Ah, for future reference: that means you need to mount (as in: kea mount) KibanaLogic.